src/lib/modernExtend.ts

```diff
@@ -1574,6 +1576,9 @@ export function iasZoneAlarm(args: IasArgs): ModernExtend {
                     if (args.zoneAttributes.includes('battery_defect')) {
                         payload = {battery_defect: (zoneStatus & (1 << 9)) > 0, ...payload};
                     }
+                    if (args.zoneAttributes.includes('hush')) {
```

Comment:
I reliase now that this option is device specific, therefore it cannot be put directly here. I think we should allow passing custom attributes to `iasZoneAlarm` instead.